app/helpers/assessment_detail_helper.rb

```diff
@@ -4,10 +4,24 @@ module AssessmentDetailHelper
   def neighbour_responses_summary_text(neighbour_responses_by_summary_tag)
     return "There are no neighbour responses" unless neighbour_responses_by_summary_tag.any?
 
-    summary_text = neighbour_responses_by_summary_tag.map do |tag, count|
-      "#{count} #{tag}"
+    summary_text = neighbour_responses_by_summary_tag.map.with_index do |(tag, count), i|
+      if i == 0
+        "There #{(count != 1) ? "are" : "is"} #{count} #{tag}#{(count != 1) ? "s" : ""}"
+      else
+        "#{count} #{tag}"
+      end
     end
 
-    "There is #{summary_text.join(", ")}."
+    "#{summary_text.join(", ")}."
+  end
+
+  def updated_neighbour_responses_summary_text(responses, detail)
+    count = if detail.created_at.nil?
+      responses.count
+    else
+      responses.where("neighbour_responses.created_at > ?", detail.updated_at).count
+    end
```

Comment:
Is the `responses` relation loaded at this point? If so it might be better to do this in memory and not use `count`